odoo/addons/base/models/ir_translation.py

```diff
@@ -523,10 +523,11 @@ def check(self, mode):
         # check for read/write access on translated field records
         fmode = 'read' if mode == 'read' else 'write'
         for mname, ids in model_ids.items():
-            records = self.env[mname].browse(ids)
-            records.check_access_rights(fmode)
-            records.check_field_access_rights(fmode, model_fields[mname])
-            records.check_access_rule(fmode)
+            records = self.env[mname].browse(ids).exists()
+            if records:
+                records.check_access_rights(fmode)
+                records.check_field_access_rights(fmode, model_fields[mname])
+                records.check_access_rule(fmode)
```

Comment:
```suggestion
            records.check_access_rights(fmode)
            records.check_field_access_rights(fmode, model_fields[mname])
            if records:
                records.check_access_rule(fmode)
```
You still want to call the `check_access_rights` as one can not import translation on a model/fields he does not have access to.